src/test/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerCaseFlagTest.java

```diff
@@ -50,4 +68,124 @@ void shouldNotConfigureCaseFlagIfFeatureFlagFalse() {
             .doesNotContain(CASEWORKER_CASE_FLAG);
     }
 
+    @Test
+    void shouldSuccessfullyAddFlagRepresentative() {
+        //Given
+        final CaseData caseData = caseData();
```

Comment:
Many test cases seem to be using same Given section. So, better to create a common private method to initiate the case data.

Reply:
Each test case has different CicCase data set based on scenario.